superset-frontend/.eslintrc.js

```diff
@@ -161,6 +162,28 @@ module.exports = {
         'react/static-property-placement': 0, // re-enable up for discussion
         'prettier/prettier': 'error',
         'file-progress/activate': 1,
+        // delete me later: temporary rules to help with migration
```

Comment:
Tracking this here https://github.com/orgs/apache/projects/315/views/1?pane=issue&itemId=53360236